.github/workflows/breaking-change-review.yml

```diff
@@ -0,0 +1,145 @@
+name: PR Review Workflow for Breaking Changes
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - labeled
+      - unlabeled
+      - synchronize
+
+jobs:
+  check_breaking_change:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for breaking change label
+        id: check_label
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const { data: labels } = await github.rest.issues.listLabelsOnIssue({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+            });
+            const hasBreakingChange = labels.some(label => label.name === 'breaking change');
+            console.log(`Has breaking change label: ${hasBreakingChange}`);
+            return hasBreakingChange;
+
+      - name: Set QA reviewers
+        if: steps.check_label.outputs.result == 'true'
+        id: set_reviewers
+        run: |
+          echo "mobile_qa=churik,yevh-berdnyk,VolodLytvynenko,pavloburykh,mariia-skrypnyk,Horupa-Olena" >> $GITHUB_OUTPUT
+          echo "desktop_qa=anastasiyaig,virginiabalducci,glitchminer,antdanchenko" >> $GITHUB_OUTPUT
+
+      - name: Request QA reviews
+        if: steps.check_label.outputs.result == 'true'
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const mobileQA = '${{ steps.set_reviewers.outputs.mobile_qa }}'.split(',');
+            const desktopQA = '${{ steps.set_reviewers.outputs.desktop_qa }}'.split(',');
+            const reviewers = [...mobileQA, ...desktopQA];
+            
+            await github.rest.pulls.requestReviewers({
```

Comment:
Might this re-request reviews, if already approved? I think we don't want such behaviour?